src/main/java/carpet/script/utils/SystemInfo.java

```diff
@@ -60,6 +64,38 @@
             WorldProperties prop = c.s.getServer().getOverworld().getLevelProperties();
             return ListValue.of(NumericValue.of(prop.getSpawnX()), NumericValue.of(prop.getSpawnY()), NumericValue.of(prop.getSpawnZ()));
         });
+
+        put("world_min_height", c-> {
```

Comment:
tbh, more practical and simpler would be to return that info for current dimension. Since prefix `world` means current save, regardless of the dimension, maybe you could use `dimension_min_height`, `dimension_max_height`, `dimension_center`, `dimension_radius` (I would separate them)